src/asmcnc/comms/router_machine.py

```diff
@@ -165,6 +168,46 @@ def __init__(self, win_serial_port, screen_manager, settings_manager, localizati
         self.TMC_motor[TMC_Y2] = motors.motor_class(TMC_Y2)
         self.TMC_motor[TMC_Z] = motors.motor_class(TMC_Z)
 
+    # disable ssh (will not start on boot)
+    def disable_ssh(self):
+        self.stop_ssh()
+        os.system('sudo systemctl disable ssh')
+
+    # enable ssh (will start on boot, only for dev use)
+    def enable_ssh(self):
+        self.start_ssh()
+        os.system('sudo systemctl enable ssh')
+
+    # toggle ssh service running or not
+    def toggle_ssh(self):
+        ssh_running = self.is_service_running('ssh')
+
+        if ssh_running:
+            return self.stop_ssh()
+
+        return self.start_ssh()
+
+    # stop ssh (doesn't change status on boot)
+    def stop_ssh(self):
+        os.system('sudo systemctl stop ssh')
+
+        return not self.is_service_running('ssh')
+
+    # start ssh (doesn't change status on boot)
+    def start_ssh(self):
+        os.system('sudo systemctl start ssh')
+
+        return self.is_service_running('ssh')
+
+    def is_service_running(self, service):
+        try:
+            with open(os.devnull, 'wb') as hide_output:
+                exit_code = subprocess.Popen(['systemctl', 'is-active', service],
+                                             stdout=hide_output, stderr=hide_output).wait()
+                return exit_code == 0
+        except:
+            log("Couldn't check status")
+
```

Comment:
All of this should be in the settings manager module, which handles this kind of stuff. This module is also the ones that's already passed to server connection module, so won't need to pass the machine module